### PR TITLE
fix: changed "Twitter" to "𝕏 (Twitter)" in README.md description section

### PR DESCRIPTION
**Pull request by BandhiyaHardik**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [ ] Have you successfully run `brew style` with your changes locally?
- [ ] Have you successfully run `brew typecheck` with your changes locally?
- [ ] Have you successfully run `brew tests` with your changes locally?

-----
This fixes https://github.com/Homebrew/brew/issues/16168

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Please consider a regular donation through [GitHub Sponsors](https://github.com/
 ## Community
 
 - [Homebrew/discussions (forum)](https://github.com/orgs/Homebrew/discussions)
-- [@MacHomebrew (Twitter)](https://twitter.com/MacHomebrew)
+- [@MacHomebrew (𝕏 (Twitter))](https://twitter.com/MacHomebrew)
 - [@homebrew@fosstodon.org (Mastodon)](https://fosstodon.org/@homebrew)
```
Comment (MikeMcQuaid) [inline README.md:34]:
```suggestion
- [@homebrew@fosstodon.org (Mastodon)](https://fosstodon.org/@homebrew)
- [@MacHomebrew (𝕏 (formerly known as Twitter))](https://twitter.com/MacHomebrew)
```

**Review by MikeMcQuaid — APPROVED** [review]
Makes sense, thanks @BandhiyaHardik. 

For future reference: please don't open issues that you plan on immediately opening a PR for ❤️

**Comment by BandhiyaHardik** [comment]
> Makes sense, thanks @BandhiyaHardik.
> 
> For future reference: please don't open issues that you plan on immediately opening a PR for ❤️

Sorry About that I am new to opensource.

I will keep this in mind.

**Comment by MikeMcQuaid** [comment]
@BandhiyaHardik No apologies needed and thanks for the PR! Policies like this vary from project to project; just letting you know ours.

**Comment by BandhiyaHardik** [comment]
> @BandhiyaHardik No apologies needed and thanks for the PR! Policies like this vary from project to project; just letting you know ours.

Yes. In Future i will keep this in mind when i contribute to brew

**Pull request merged**